bindgen/clang.rs

```diff
@@ -2351,7 +2351,7 @@ impl EvalResult {
 
         if unsafe { clang_EvalResult_isUnsignedInt(self.x) } != 0 {
             let value = unsafe { clang_EvalResult_getAsUnsigned(self.x) };
-            if value > i64::MAX as c_ulonglong {
+            if value > u64::MAX as c_ulonglong {
```

Comment:
So to be clear, we're relying on returning `Some(value as i64)` which is potentially negative since it overflows. And then presumably we cast it back when generating...

That feels a bit sketchy, can you add some comments? Also are the rust conversions well defined? In C++ this would be UB for sure I think.